lib/streamlit/elements/widgets/select_slider.py

```diff
@@ -83,7 +80,7 @@ def deserialize(
 
         # The widget always returns floats, so convert to ints before indexing
         return_value: tuple[T, T] = cast(
-            Tuple[T, T],
+            tuple[T, T],
```

Comment:
@lukasmasuch I'm a little confused by this, because there are definitely other places in the code base where we use Tuple, and I have even seen places where ruff will tell me I have to use the capital version and not the lowercase version or it won't pass the lint check...

Reply:
See https://github.com/Asaurus1/streamlit/blob/00e115c288b4ecf171257f0d2778de96a5566d49/lib/streamlit/elements/widgets/slider.py#L313 where it will yell at me if I return a Tuple instead of a tuple.

Reply:
Yep, `from __future__ import annotations` makes it possible to use the new type annotations in older Python versions for most cases. But there seems to be a few exceptions, e.g. when used in a `cast` or in a TypeAlias (as far as I know).